core/src/main/scala/org/apache/spark/h2o/utils/H2OContextRestAPIUtils.scala

```diff
@@ -52,6 +52,22 @@ trait H2OContextRestAPIUtils extends H2OContextUtils {
     getNodes(cloudV3)
   }
 
+  def verifyWebOpen(nodes: Array[NodeDesc], conf: H2OConf): Unit = {
+    val nodesWithoutWeb = nodes.flatMap { node =>
+      try {
+        getCloudInfoFromNode(node, conf)
+        None
+      } catch {
+        case _: H2OClusterNodeNotReachableException => Some(node)
```

Comment:
If a node doesn't have web enabled, is the exception thrown immediately or there any timeout? I'm wondering about the case when a cluster of hundreds nodes have all web endpoints disabled.

Reply:
The exception is thrown right away after we discover the endpoint does not exist -> so I would say almost immediately. 

We could throw exception right away after we discover first node without web enabled and improve the exception message "Please, make sure that the rest of the nodes have web enabled as well".

What I like about printing the full list of non-available nodes is that we have all information at one place.

What do you think?

Reply:
I like as it's right now. I just wasn't sure how much time this could take. If it's immediate then it's fine ;-)

Reply:
Cool, thx!